smart_open/gcs.py

```diff
@@ -39,17 +40,19 @@
 DEFAULT_BUFFER_SIZE = 256 * 1024
 """Default buffer size for working with GCS"""
 
-_UPLOAD_INCOMPLETE_STATUS_CODE = 308
+_UPLOAD_INCOMPLETE_STATUS_CODES = (308, )
```

Comment:
Why this change?

Reply:
To simplify the check against expected values in the upload_part function.